bamboo/summary.py

```diff
@@ -70,8 +77,13 @@ def pr(line):
             pr("<table>")
             first = True
             for repo in repos:
-                pr('<tr {4}> <td class="col1"><a href="{0}">{1}</a></td> <td class="col2"><a href="{2}"><code>{3}</code></a><td> </tr>'.format(
-                    repo_link(repo_names[repo]), repo_names[repo], link(repo_names[repo], shas[repo]), short(shas[repo]), "" if first else 'class="separate"'))
+                separator = "" if first else 'class="separate"'
```

Comment:
could use enumerate here

Reply:
Yes, after having tried it out, I think that is nicer.